app/src/main/java/is/genki/bigredapp/android/DiningListFragment.java

```diff
@@ -30,7 +31,7 @@
 public class DiningListFragment extends Fragment {
 
     private static final String BASE_URL = "http://redapi-tious.rhcloud.com/dining";
-    private static final String[] MEALS_LIST = {"Breakfast", "Lunch", "Dinner"};
+    public static final String[] MEALS_LIST = {"Breakfast", "Lunch", "Dinner"};
```

Comment:
Oh good point. I found the source for RedAPI @ https://github.com/mrkev/Iroh and https://github.com/mrkev/redapi. Looking at the repos... maybe it wouldn't be a bad idea to set up a new repository, consulting/working with Kevin, to provide the backend for this app, that could be then used for other platforms too. Just tossing the idea out there.